docs/kepler_model_server/pipeline.md

```diff
@@ -1,29 +1,20 @@
 # Training Pipeline
 
-As shown in the below figure, a power model can be attributed by the path it was trained. Training pipeline is an abstract of power model training that applies a set of learning methods to a different combination of energy source labels, available metrics, and idle/control plane power isolation choices to each specific group of nodes/machines. 
+Model server serves extensive choices of power models. Training pipeline is an abstract of power model training that applies a set of learning methods to a different combination of energy sources, power isolation methods, available energy-related metrics.  
```

Comment:
`Model server serves extensive choices of power models.`
->
`Model server can provide various power models for different context and learning methods.`